### PR TITLE
EAS Build

### PR DESCRIPTION
**Pull request by JUNIOR140889**
### Pull request type

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

---

### Description

Add EAS build basic config

---

#### Jira board reference

- [Add EAS Build](https://www.notion.so/rootstrap/Add-EAS-Build-bb9b59cad5b94fcdbb7269608ef0b950)

---

#### Notes:

-

---

#### Tasks:

- [X] Tested on IOS
- [ ] Tested on Android
- [ ] Tested on a small device
- [ ] Tested on a real device
- [ ] Tested all flows related with this PR changes
- [ ] Tested accessibility
- [ ] Added tests

---

#### Preview

Screenshot 2024-05-24 at 9 11 16 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,17 @@ npx pod-install
 1. Create a new app in firebase with the package with the suffix of the flavor you will run
 2. Download the `google-services.json` and move it to the `./android/app` directory
 
+## EAS Build
+
```
Comment (fernandatoledo) [inline README.md:52]: WDYT about adding the steps to install eas locally can be added to the readme with the link to the official doc?

File: ios/firebase/GoogleService-Info-dev.plist
Comment (fernandatoledo) [inline ios/firebase/GoogleService-Info-dev.plist:1]: I think this files might have been added by mistake to the repo?

Comment (JUNIOR140889) [reply ios/firebase/GoogleService-Info-dev.plist:1]: It's added to support multiple bundle identifier to firebase using a script to copy and link the corresponding .plist in each build.

File: README.md (hunk 1 above)
Comment (JUNIOR140889) [reply README.md:52]: Done

File: app.json [hunk 2]
```diff
@@ -1,4 +1,10 @@
 {
   "name": "ReactNativeBase",
-  "displayName": "ReactNativeBase"
-}
\ No newline at end of file
+  "displayName": "ReactNativeBase",
+  "expo": {
+    "extra": {
+      "eas": {
+      }
```
Comment (guillermomachado) [inline app.json:7]: Should we add the `projectId` placeholder here?

**Review by guillermomachado — APPROVED** [review]

File: app.json (hunk 2 above)
Comment (JUNIOR140889) [reply app.json:7]: It is added automatically when you execute the EAS build command and add the project to your Expo account.

**Pull request closed without merging**